packages/pipeline-editor/src/CustomFormControls/index.ts

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+/* istanbul ignore file */
```

Comment:
Why is this here? Jest should automatically ignore this right?

Reply:
We don't import this file into any of our tests so it's shows up as uncovered even though it's just exports. Jest is collecting coverage from all ts files except for the root index.ts file (this file isn't in the root src folder)